src/assets/YAML/default/BuildAndDeployment/Deployment.yaml

```diff
@@ -52,8 +52,7 @@ Build and Deployment:
     Defined deployment process:
       risk: Deployments without a defined process are error prone thus allowing old
         or untested artifact to be deployed.
-      measure: A defined deployment process significantly lowers the likelihood of
-        errors during the deployment phase.
+      measure: A defined deployment process ensures that .
```

Comment:
this sentence ends out of the blue, is that intended?

Reply:
draft PR ;)